.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: c++
+# command to install dependencies
+before_install:
+  # Dependencies from  <http://mixxx.org/wiki/doku.php/compiling_on_linux>
+  - sudo apt-get install git scons libqt4-dev libqt4-sql-sqlite libportmidi-dev libshout3-dev libtag1-dev libprotobuf-dev protobuf-compiler libvamp-hostsdk3 vamp-plugin-sdk libusb-1.0-0-dev libfftw3-dev libmad0-dev portaudio19-dev libchromaprint-dev librubberband-dev libsqlite3-dev libsndfile1-dev libflac-dev libid3tag0-dev
+  # Virtual X
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+install:
+  - scons
```

Comment:
Tried it and found that `scons tests=1` does not build the tests. `scons test=1` on the other hand does, cf. <https://travis-ci.org/nschloe/mixxx/builds/48262061>.